src/Common/tests/System/RealFormatterTestsBase.netcoreapp.cs

```diff
@@ -0,0 +1,326 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Tests
+{
+    public abstract class RealFormatterTestsBase
+    {
+        // The following methods need to be implemented for the tests to run:
+        protected abstract string InvariantToStringDouble(double d, string format);
+        protected abstract string InvariantToStringSingle(float f, string format);
+
+        // Skipped due to https://github.com/dotnet/corefx/issues/35566
```

Comment:
Instead of commenting all of this (and technically you would only need to comment out the [Theory]), please use ActiveIssue instead.

Reply:
Fixed.